presto-native-execution/presto_cpp/main/TaskManager.h

```diff
@@ -154,8 +154,9 @@ class TaskManager {
       std::vector<velox::exec::Task::OpCallInfo>& stuckOpCalls) const;
 
   /// Build directory path for spilling for the given task.
-  /// Always returns non-empty string.
-  static std::string buildTaskSpillDirectoryPath(
+  /// Always returns tuple of non-empty string containing the spill directory
```

Comment:
```
the task spill directory and date spill directory which is parent directory of task spill directory.
```
Not sure if we want to return the two as the last two output parameters of buildTaskSpillDirectoryPath?
```
std::string& taskSpillDirPath,
std::string& dateSpillDirPath);
```